src/test/a11y/constants.js

```diff
@@ -0,0 +1,48 @@
+function randomCharFromChars (chars) {
```

Comment:
This file was the remnants of what was needed from constants.js file in the common directory.